lib/drivers/mongo/transpiler/build-conditions.js

```diff
@@ -44,6 +46,21 @@ module.exports = () => {
     Object.assign(acc, {[OPERATORS[key]]: value});
   };
 
+  const isRegex = value => {
+    if (!isObject(value)) return false;
+    return value.hasOwnProperty('regex');
+  };
+
+  const swapRegex = (acc, val, key) => {
+    const {regex, case_sensitive: caseSensitive} = val;
+    const value = {$regex: `^${regex}`};
```

Comment:
Trucha, haciendo esto cuando tengas textos con saltos de línea tal vez no tenga el comportamiento esperado
https://docs.mongodb.com/manual/reference/operator/query/regex/#multiline-match-for-lines-starting-with-specified-pattern